clang/test/Driver/sycl.c

```diff
@@ -75,6 +75,8 @@
 // -fsycl-force-inline-kernel-lambda
 // RUN: %clangxx -### -fsycl-device-only -fno-sycl-force-inline-kernel-lambda  %s 2>&1 | FileCheck %s --check-prefix=CHECK-NOT-INLINE
 // RUN: %clang_cl -### -fsycl-device-only -fno-sycl-force-inline-kernel-lambda  %s 2>&1 | FileCheck %s --check-prefix=CHECK-NOT-INLINE
+// RUN: %clangxx -### -fsycl-device-only -O0 %s 2>&1 | FileCheck %s --check-prefix=CHECK-NOT-INLINE
+// RUN: %clang_cl -### -fsycl-device-only -Od %s 2>&1 | FileCheck %s --check-prefix=CHECK-NOT-INLINE
```

Comment:
Can we add a negative check to be sure `-fno-sycl-force-intel-kernel-lambda` isn't being set with optimizations are enabled?

Reply:
Sure, what option(s) should I use?  -O1?

Reply:
`-O1` or `-O2` should do the trick.

Reply:
Done.